src/olympia/addons/serializers.py

```diff
@@ -152,8 +156,16 @@ def fake_object(self, data):
         self._attach_fields(
             obj, data,
             ('average_daily_users', 'bayesian_rating', 'created',
-             'default_locale', 'guid', 'hotness', 'is_listed', 'last_updated',
-             'public_stats', 'slug', 'status', 'type', 'weekly_downloads'))
+             'default_locale', 'guid', 'hotness', 'icon_type', 'is_listed',
+             'last_updated', 'modified', 'public_stats', 'slug', 'status',
+             'type', 'weekly_downloads'))
+
+        # Temporary hack to make sure all add-ons have a modified date when
+        # serializing, to avoid errors when calling get_icon_url().
+        # Remove once all add-ons have been reindexed at least once since the
```

Comment:
I felt it was unnecessary since I'm spending most of my time looking at this file these days :)